treeherder/config/settings.py

```diff
@@ -169,7 +163,6 @@
 # Caches
 REDIS_URL = env('REDIS_URL', default='redis://localhost:6379')
 if connection_should_use_tls(REDIS_URL):
-    # Connect using TLS on Heroku.
     REDIS_URL = get_tls_redis_url(REDIS_URL)
```

Comment:
I don't know if this is needed for use outside of heroku. The function its calling is this: https://github.com/mozilla/treeherder/blob/master/treeherder/config/utils.py#L12 and it all looks very heroku specific. @oremj please advise.

Reply:
I think it would be better just to replace this with an option to connect via SSL rather than trying to do automatic detection.

Reply:
What I mean to ask is whether we should be disabling `url.args['ssl_cert_reqs'] = 'none'` for redis, per https://github.com/mozilla/treeherder/blob/master/treeherder/config/utils.py#L34. My guess is no, but please correct me if I'm wrong.

Reply:
@oremj Could you please weigh in here or else direct me to something else? :)

Reply:
@jbuck Can you help with this so I can get this pr merged?